### PR TITLE
Add marketplace list command

### PR DESCRIPTION
**Pull request by scottcrawford03**
**Background**
Add 1-click command to doctl to list all available 1-click slugs and type.

**Changes**
* Adds the functionality to type `doctl 1-click list` which will output all 1-click apps by type. to narrow it down further you can do `doctl 1-click list --type droplet`
* Add tests

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,7 +15,7 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/gogo/protobuf v1.3.1 // indirect
 	github.com/golang/mock v1.4.0
-	github.com/golang/protobuf v1.4.0 // indirect
+	github.com/golang/protobuf v1.4.2 // indirect
```
Comment (scottcrawford03) [inline go.mod:18]: @bentranter is this version bump okay?

Comment (bentranter) [reply go.mod:18]: Yup, these look okay – I just realized some of these were pulled in as a result of different godo changes than yours.

File: commands/1_clicks.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
```
Comment (bentranter) [inline commands/1_clicks.go:2]: I know this is from a copy-paste, but could you switch the copyright year here to 2020?

File: commands/displayers/1_click.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
```
Comment (bentranter) [inline commands/displayers/1_click.go:2]: Same here with the copyright year.

File: do/1_clicks.go [hunk 4]
```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
```
Comment (bentranter) [inline do/1_clicks.go:2]: And here as well.

File: do/1_clicks.go [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+package do
+
+import (
+	"context"
+
+	"github.com/digitalocean/godo"
+)
+
+// OneClickService is the godo VPCsService interface.
```
Comment (bentranter) [inline do/1_clicks.go:22]: Is mentioning the VPCsService here a copy-paste error?

File: commands/1_clicks.go [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+)
+
+// OneClicks creates the 1-click command.
+func OneClicks() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "1-click",
+			Short: "Display commands that pertain to 1-click applications",
+			Long: `The commands under ` + "`" + `doctl 1-click` + "`" + ` are for interacting with DigitalOcean 1-Click applications.
+With the 1-click command, you can list 1-Click applications.`,
```
Comment (andrewsomething) [inline commands/1_clicks.go:29]:
```suggestion
			Long:  "The commands under `doctl 1-click` are for interacting with DigitalOcean 1-Click applications.",
```

One tiny nitpick: I'd remove the second line here.

File: do/1_clicks.go (hunk 5 above)
Comment (scottcrawford03) [reply do/1_clicks.go:22]: yes

**Pull request merged**